### PR TITLE
x1139 update amplification operation

### PR DESCRIPTION
**Pull request by sabrine33**

### CODE REVIEW

**Review by khelwood — APPROVED** [review]

**Comment by khelwood** [comment]
@sabrine33 This pr fails cypress tests every time they run.
```none
5) Visium QC Page
       On Visium QCType as Amplification
         On Save
           When there is no server error
             "after all" hook for "shows a success message":
     CypressError: `cy.click()` failed because it requires a DOM element.
```
Can you take a look at it please?

**Comment by sabrine33** [comment]
I will have a look, thanks

**Comment by khelwood** [comment]
@sabrine33 That might have helped, but it's still failing in `pages/visiumQC`

**Comment by khelwood** [comment]
@sabrine33 Thanks for fixing the tests. :+1: Can you fix the merge conflicts please?

**Comment by sabrine33** [comment]
@khelwood, I'd rather wait until @seenanair fixes the storybook issue in the CI.

File: cypress/e2e/pages/visiumQC.cy.ts [hunk 1]
```diff
@@ -58,8 +58,10 @@ describe('Visium QC Page', () => {
       });
       it('has all slots as passed', () => {
         cy.findAllByTestId('passIcon').then(($passIcons) => {
+          debugger;
```
Comment (khelwood) [inline cypress/e2e/pages/visiumQC.cy.ts:61]: Please remember to remove this debugger

**Pull request merged**